src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx

```diff
@@ -20,29 +30,85 @@ export interface AdminSpaceCardProps {
 }
 
 export const AdminSpaceCard: React.FC<AdminSpaceCardProps> = ({ venue }) => {
+  const [validBannerImageUrl] = useValidImage(
+    venue?.config?.landingPageConfig.bannerImageUrl,
+    DEFAULT_VENUE_BANNER_COLOR
+  );
+
   const backgroundStyle = useCss({
-    "background-image": `url(${venue.mapBackgroundImageUrl})`,
+    background: `linear-gradient(
```

Comment:
```suggestion
    // @debt There should be a way to move the rgba() functions to SCSS and use $color-constant instead of literals
    background: `linear-gradient(
```

Reply:
done 0ab68ac